player/command.c

```diff
@@ -5085,6 +5085,12 @@ static void cmd_stop(void *p)
 
     if (!(flags & 1))
         playlist_clear(mpctx->playlist);
+
+    if (mpctx->opts->player_idle_mode < 2 &&
+        mpctx->opts->position_save_on_quit) {
+      mp_write_watch_later_conf(mpctx);
+    }
```

Comment:
There are still 2 style issues: indentation, `{` placement. See contribute.md.

Reply:
Thanks for pointing out those two, should've read contribute.md more thoroughly. I believe style issues are resolved by now.